src/components/SearchBox/SearchBox.js

```diff
@@ -0,0 +1,213 @@
+import React, { Component } from 'preact-compat';
+import PropTypes from 'prop-types';
+import noop from 'lodash/noop';
```

Comment:
I don't think we use this one anywhere else; so in that case just writing it in full once seems better

Reply:
We do – we can remove this dependency in another pass.